apps/pie-docs/src/components/toast/overview/overview.md

```diff
@@ -0,0 +1,376 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Toast
+    order: 1
+eleventyComputed:
+    priorityQueue: "{% include './priority-queue.json' %}"
+shouldShowContents: true
+permalink: components/toast/
+---
+
+## Overview
+
+Toasts are used to inform users about processes within a system. They can also provide contextual feedback. They appear temporarily at the bottom-left of the screen and disappear after a certain time period or when the user performs an action to remove them.
```

Comment:
Just checking; is it always bottom left, even for RTL?

Reply:
updated to remove the left part.